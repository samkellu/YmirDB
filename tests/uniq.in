set a 1 1 1 2 2 3 3 3 3 3 4 4
set b 1 1 a a a 22 22 22
uniq a
uniq b
list entries
bye
