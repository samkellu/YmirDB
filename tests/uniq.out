> ok

> ok

> ok

> simple entry only

> b [1 1 a a a 22 22 22]
a [1 2 3 4]

> bye
